service/internal/telemetrylogs/logger.go

```diff
@@ -45,5 +55,17 @@ func NewLogger(cfg config.ServiceTelemetryLogs, options []zap.Option) (*zap.Logg
 	if err != nil {
 		return nil, err
 	}
+
 	return logger, nil
 }
+
+func SetGRPCLogger(logger *zap.Logger, loglevel zapcore.Level) {
+	glogger := logger.WithOptions(zap.WrapCore(func(core zapcore.Core) zapcore.Core {
+		if loglevel == zap.InfoLevel {
+			c, _ := zapcore.NewIncreaseLevelCore(core, zap.WarnLevel)
```

Comment:
I guess this deserves an explanation :-)

Reply:
Sorry, I meant that the code deserves a comment with this explanation.

Reply:
Added new commit! thank you :)